test/Microsoft.TemplateEngine.Edge.UnitTests/LocalizationTests.cs

```diff
@@ -148,10 +148,12 @@ public void TestLocalizedSymbolDescription(string locale, string expectedDescrip
         }
 
         [Theory]
-        [InlineData("de", "name_de-DE:äÄßöÖüÜ")]
+        [InlineData("de", "name")]
         [InlineData("de-de", "name_de-DE:äÄßöÖüÜ")]
-        [InlineData("de-Au", "name_de-DE:äÄßöÖüÜ")]
-        [InlineData("de-AU", "name_de-DE:äÄßöÖüÜ")]
+        [InlineData("de-AT", "name")]
```

Comment:
This seems like wrong change, de-AT should be translated to german, no?

Reply:
In test template localization file is de-de. Imo we should not fallback in case file explicitly mentions country/region. The behavior of resource files is as this.